src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -574,12 +584,36 @@ public void PublishUnwindInfo(ObjectNode node)
             }
         }
 
+        private static ReadOnlySpan<byte> DwarfArm64EmptyFrame => new byte[]
+        {
+            0x04, 0x00, 0xFF, 0xFF, 0x10, 0x00, 0x00, 0x00,
+            0x04, 0x02, 0x1D, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x04, 0x02, 0x1E, 0x00, 0x08, 0x00, 0x00, 0x00,
+            0x08, 0x01, 0x1D, 0x00, 0x00, 0x00, 0x00, 0x00
+        };
```

Comment:
Represents the following DWARF code:
```
  DW_CFA_advance_loc: 4
  DW_CFA_def_cfa_offset: +16
  DW_CFA_offset: W29 -16
  DW_CFA_offset: W30 -8
  DW_CFA_advance_loc: 4
  DW_CFA_def_cfa_register: W29
```

for this ARM64 frame:
```
stp x29, x30, [sp, #-10]!
mov x29, sp
…
ldr x29, x30, [sp], #0x10
ret
```

Reply:
Could you please turn this into a source code comment?

Reply:
> ldr x29, x30, [sp], #0x10

typo: should be `ldp`

Reply:
Done.